mavlink/modules/flight_controller.py

```diff
@@ -95,3 +95,21 @@ def get_home_location(self, timeout: float) -> "tuple[bool, drone_odometry.Drone
             return False, None
 
         return True, location
+
+    def drone_travelling_to_final_waypoint(self) -> "tuple[bool, bool | None]":
```

Comment:
Prefix `is_` to name.

May also want to remove/rename `travelling` , since the drone is not necessarily travelling to that waypoint (only that the drone's destination is that waypoint, but the drone could be in manual mode).